static/css/components/searchbox.less

```
@@ -0,0 +1,33 @@
+.searchbox {
+  background-color: @white;
+  border: 1px solid @dark-beige;
+  border-radius: 5px;
+  position: relative;
+  display: inline-flex;
+  align-items: center;
+  padding: 2px;
+  padding-left: 5px;
+  width: 50%;
+  height: 30px;
+
+  &__btn-icon {
+    background: url(/static/images/search-icon.svg) center no-repeat;
+    width: 35px;
+    cursor: pointer;
+    border: 0;
+  }
+
+  &__input {
+    margin: 0 !important;
+    border: 0;
+    background: transparent;
+    color: @grey !important;
+    flex-grow: 1;
+    font-size: 1em !important;
+    min-width: 0;
+  }
+
+  @media (max-width: @width-breakpoint-mobile) {
```

Ah it seems like previously we went full width at the table, so bumping this up.
```suggestion
  @media (max-width: @width-breakpoint-tablet) {
```